podcast/2.6.md

```diff
@@ -0,0 +1,741 @@
+---
+title: "Designing delightful docs with Orta Therox, engineer on Microsoft's TypeScript compiler team"
+publishDate: 2021-09-14T10:00-07:00
+tags: [podcast]
+slug: orta-therox
+published: true
+---
+
+<!-- START AUDIO -->
+
+<audio className="object-center" src="https://www.buzzsprout.com/1097978/9110277-orta-therox-typescript-compiler-engineer-at-microsoft.mp3" controls preload="none"></audio>
+
+<!-- END AUDIO -->
+
+<!-- START GUESTS -->
+<span>
+Orta Therox, Beyang Liu
+</span>
+<!-- END GUESTS -->
+
+<!-- START SUMMARY -->
+
+How do you design software documentation and websites that both intrigue and educate? As a contributor to popular projects like React Native, Jest, Prettier, and TypeScript, Orta Therox has prioritized design for visual engagement, accessibility, and learning. In this episode, Orta talks about the importance of engaging docs, how experimentation fuels learning and engineering in TypeScript, and how developers can write better code examples with Shiki Twoslash, a project he developed and designed. Along the way, Orta also shares his own story of getting into code and the odd way he was hired on Microsoft's TypeScript compiler team.
+
+Click the audio player below to listen, or click [here]() to watch the video.
+
+<!-- END SUMMARY -->
+
+<!-- START SHOWNOTES -->
+
+React: https://reactjs.org/
+
+React Native: https://reactnative.dev/
+
+Relay: https://relay.dev/
+
+Apollo: https://www.apollographql.com/
+
+prettier: https://prettier.io/
+
+Jest: https://jestjs.io/
+
+GraphQL: https://graphql.org/
+
+GraphQL Codegen: https://www.graphql-code-generator.com/
+
+Docusaurus: https://docusaurus.io/
+
+esbuild: https://esbuild.github.io/
+
+CocoaPods: https://cocoapods.org/
+
+TypeScript: https://www.typescriptlang.org/
+
+Flow: https://flow.org/
+
+Super Mario Maker: https://en.wikipedia.org/wiki/Super_Mario_Maker
+
+Quicksilver: https://qsapp.com/
+
+Colloquy: https://github.com/colloquy/colloquy
+
+Adium: https://adium.im/
+
+TypeScript Playground: https://www.typescriptlang.org/play
+
+TypeScript Docs: https://www.typescriptlang.org/docs/
+
+Swift Playgrounds: https://www.apple.com/swift/playgrounds/
+
+Shiki Twoslash: https://github.com/shikijs/twoslash
```

Comment:
@csbailey5t The original link was 404ing, is this one suitable?

Reply:
There was a missing "s"; it should be https://shikijs.github.io/twoslash/ which is preferable to the github link

Reply:
```suggestion
Shiki Twoslash: https://shikijs.github.io/twoslash/
```